test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -6,348 +6,358 @@ const {
-          await driver.clickElement({ text: 'Confirm', tag: 'button' });
-
-          await driver.switchToWindow(windowHandles.dapp);
-
-          const tokenContractAddress = await driver.waitForSelector({
-            css: '#tokenAddress',
-            text: '0x',
-          });
-          const tokenAddress = await tokenContractAddress.getText();
-
-          // imports custom token from extension
-          await driver.switchToWindow(windowHandles.extension);
-          await driver.clickElement(`[data-testid="home__asset-tab"]`);
-          await driver.clickElement({ tag: 'button', text: 'Assets' });
-
-          await driver.clickElement({ text: 'import tokens', tag: 'a' });
-          await driver.clickElement({
-            text: 'Custom token',
-            tag: 'button',
-          });
-          await driver.waitForSelector('#custom-address');
-          await driver.fill('#custom-address', tokenAddress);
-          await driver.waitForSelector('#custom-symbol');
-          await driver.waitForSelector('#custom-decimals');
-          await driver.delay(2000);
-
-          await driver.clickElement({
-            text: 'Add custom token',
-            tag: 'button',
-          });
-
-          await driver.delay(2000);
-          await driver.clickElement({
-            text: 'Import tokens',
-            tag: 'button',
-          });
-
-          // renders balance for newly created token
-          await driver.waitForSelector('.app-header__logo-container');
-          await driver.clickElement('.app-header__logo-container');
-          await driver.clickElement({ tag: 'button', text: 'Assets' });
-          const asset = await driver.waitForSelector({
-            css: '.asset-list-item__token-value',
-            text: '10',
-          });
-          assert.equal(await asset.getText(), '10');
-        },
-      );
-    });
+const { SMART_CONTRACTS } = require('../seeder/smart-contracts');
+
+describe('Create token, approve token and approve token without gas', function () {
+  const smartContract = SMART_CONTRACTS.HST;
+  const ganacheOptions = {
+    accounts: [
+      {
+        secretKey:
+          '0x7C9529A67102755B7E6102D6D950AC5D5863C98713805CEC576B945B15B71EAC',
+        balance: convertToHexValue(25000000000000000000),
+      },
+    ],
+  };
+
+  it('creates, imports and renders the balance for the new token', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: 'connected-state',
+        ganacheOptions,
+        smartContract,
+        failOnConsoleError: false,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await driver.fill('#password', 'correct horse battery staple');
+        await driver.press('#password', driver.Key.ENTER);
+
+        // create token
+        await driver.openNewPage(`http://127.0.0.1:8080/`);
+        await driver.clickElement({ text: 'Create Token', tag: 'button' });
```

Comment:
You shouldn't need to create the token, as its already created using the seeder

Reply:
similar to the tests below

Reply:
Done